README.md

```diff
@@ -27,6 +27,20 @@ sudo apt-get update
 sudo apt-get install lazygit
 ```
 
+### Arch Linux
+Packages for Arch Linux are available via AUR (Arch User Repository).
+
+There are two packages. The stable which is built with the last releases and the git version which takes the last commit and build based on.
```

Comment:
there are some grammar errors here, and I think it's missing a few words in the second sentence?